generated/nidaqmx/constants.py

```diff
@@ -169,12 +169,12 @@ class CalibrationTerminalConfig(Enum):
 
 
 class ChannelType(Enum):
-    ANALOG_INPUT = 10100  #: Analog input channel.
-    ANALOG_OUTPUT = 10102  #: Analog output channel.
-    DIGITAL_INPUT = 10151  #: Digital input channel.
-    DIGITAL_OUTPUT = 10153  #: Digital output channel.
-    COUNTER_INPUT = 10131  #: Counter input channel.
-    COUNTER_OUTPUT = 10132  #: Counter output channel.
+    AI = 10100  #: Analog input channel.
+    AO = 10102  #: Analog output channel.
+    DI = 10151  #: Digital input channel.
+    DO = 10153  #: Digital output channel.
+    CI = 10131  #: Counter input channel.
+    CO = 10132  #: Counter output channel.
```

Comment:
I dislike these changes. I prefer fully spelled out.

Reply:
Updated enum values as in expected constants.py